application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -112,4 +112,11 @@ private RestAction<Message> sendHelperHeadsUp(ThreadChannel threadChannel) {
         return threadChannel.sendMessage(headsUpWithoutRole)
             .flatMap(message -> message.editMessage(headsUpWithRole));
     }
+
+    private String getEmbedOrRawMessage(Message message) {
+        if (message.getEmbeds().isEmpty()) {
+            return message.getContentRaw();
+        }
+        return message.getEmbeds().get(0).getDescription();
```

Comment:
could u add together multi-embed content please? just a bit of thinking-ahead, and bc its so easy to do 😃

Reply:
Is it possible to have multiple embeds in one post? Also you mean like iterate over the list and concat all the descriptions?

Reply:
> Also you mean like iterate over the list and concat all the descriptions?

Yeah. instead of only taking the content of the first, iterate all and just `+` it together.